apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -61,7 +61,7 @@ class PageTemplateModal extends Component {
 						: null;
 					this.setState( { blocks: { ...this.state.blocks, [ slug ]: parsedBlocks } } );
 				} ).bind( null, templates[ i ] ),
-				500 * i
+				50 * i
```

Comment:
I'd be for capturing this value in a well-named variable to make the code self-documenting.

Reply:
Adding. No guarantee with _well-named_ 😅